backend/src/database/repositories/tenantRepository.ts

```diff
@@ -239,6 +245,15 @@ class TenantRepository {
   }
 
   static async findById(id, options: IRepositoryOptions) {
+    const redis = await createRedisClient(true)
+    const tenantId = `tenant:${id}`
+
+    const tenantCache = new RedisCache(tenantId, redis)
```

Comment:
`RedisCache` first parameter is redis directory name. It should be something common to all keys in there.
My suggestion would be just a string `tenants` and in redis this will appear as `cache_tenants` directory. Inside you can have key=value pairs be `tenantId = JSON string of tenant object`.